nodeJS/express/views.md

```diff
@@ -147,48 +161,37 @@ You can insert this component into another EJS file like so:
 </html>
 ```
 
-This can be used to include headers and footers in all of your pages, for example.
-
-Note that the navbar expects a `links` value. To pass this data into the navbar, you can pass it when rendering `index.ejs` which contains the navbar. Modify `app.js` such that a `links` object is defined and passed into the `render` function in the `"/"` route handler:
-
-```javascript
-// app.js
-const links = [
-  { href: "/", text: "Home" },
-  { href: "about", text: "About" },
-];
+This will include the navbar with links in `index.ejs`. Reusable templates can be used to include headers and footers in all of your pages.
 
-app.get("/", (req, res) => {
-  res.render("index", { links: links });
-});
-```
+Note the use of the raw output tag `<%-` with the `include` which is used to avoid double-escaping the HTML output.
 
-Here's another example of how to use `includes` to dynamically render a list of variables:
+Let's use `includes` to dynamically render a list of variables a different way. Add `const users = ["Bob", "Bill", "Biff"];` to `app.js` just below the `links` variable. Then create a new view called `user.ejs` in the `views` directory and add the following `<li><%= user %></li>` line. Finally add the following block to `index.ejs`:
```

Comment:
Nit: drop the `s` from `includes` (3rd word)

Non-nit: the `app.js` and `user.ejs` examples should be separate fenced code blocks like all the others, rather than inline as current. The consistency makes it easier to follow with the same importance to each new part of code. The inlines feel more like "quick references to an prior example" sort of thing. You **must** also instruct to pass the `users` array to `index.ejs` when rendering:

```js
app.get("/", (req, res) => {
  res.render("index", { links: links, users: users });
});
```

(Additional definite nit, non-blocking - what about using `Rose` and `Cake` instead of Bill and Bob? :eyes:)

Reply:
The Biff was accidental and needed for the alliteration, but including Rose and Cake would definitely be nice!

Reply:
The alliteration was nice but thought we might as well name drop a few more people :P